great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -228,21 +238,6 @@ def rules(self) -> Optional[List[Rule]]:
         """
         pass
 
-    @abstractmethod
```

Comment:
@NathanFarmer Just making sure that `def plot()` will be an abstract method in `DataAssistantResult` (if it already is, please ignore this comment).  Thanks!